### DISCUSSION TITLE
Wordpreess Güvenliği için Önemli Noktalar

### DISCUSSION THREAD

**Discussion by msalihllc** [WordPress/developer-blog-content#319] [Topic Ideas]
Wordpress Güvenlik Önlemleri için Önemli Olduğunu Düşündüğüm Noktalar

1-) Tema Seçimi : Temaların güvenliğini kontrol etmek için eklenti > [Theme Check – WordPress plugin](https://wordpress.org/plugins/theme-check/)
2-) define( 'DISALLOW_FILE_EDIT', true );    // WordPress'te Dosya Düzenleme Ayarına İzin Verme
3-) /wp-content/mu-plugins              // Klasöründeki dosyalar otomatik olarak çalıştırılırlar. 
4-) Datebase _wp ön eklerinin değişmesi sağlanabilir. // Dikkatli olmak gerekir.
5-) add_filter('xmlrpc_enabled', ‘__return_false');  //      WordPress'in uzaktan yönetim özellikleri sunmasına yardımcı olurken, potansiyel güvenlik riskleri nedeniyle dikkatle değerlendirilmelidir. Güvenliği artırmak ve performans sorunlarını önlemek için, XML-RPC'yi devre dışı bırakmak genellikle iyi bir fikirdir.
6-) <Files *.php>
    Deny from all              //.htaccess içine php yürütmelerini devre dışı bırakma
</Files> 
7-) Options - Indexes    //Dizin indexlemeyi ve göz atmayı devre dışı bırakma. - Dizin Taramasını Engelleme

**Comment by bph** [comment]
English via Google

Points I think are important for WordPress security measures

1-) theme selection: Plugin to check the security of themes> Theme Check – WordPress plugin
2-) define( 'DISALLOW_FILE_EDIT', true ); // Allowing File Editing Settings in WordPress
3-) Files in the /wp-content/mu-plugins // folder are automatically run.
4-) Datebase _wp prefixes can be changed. // Care should be taken.
5-) add_filter('xmlrpc_enabled', ‘__return_false'); // While it helps WordPress provide remote management features, it should be carefully evaluated due to potential security risks. To increase security and prevent performance issues, disabling XML-RPC is generally a good idea.
 6-) <Files *.php>
Deny from all //Disable php executions in .htaccess

7-) Options - Indexes //Disable indexing and browsing of directories. - Prevent Directory Scanning

**Comment by bph** [comment]
I am closing this as our language for now is English. and I don't feel this is well described topic suggestion.
